Makefile

```diff
@@ -380,6 +380,11 @@ install-kubernetes-tools: build/toolchain/bin/kubectl$(EXE_EXTENSION) build/tool
 install-web-tools: build/toolchain/bin/hugo$(EXE_EXTENSION) build/toolchain/bin/htmltest$(EXE_EXTENSION) build/toolchain/nodejs/
 install-protoc-tools: build/toolchain/bin/protoc$(EXE_EXTENSION) build/toolchain/bin/protoc-gen-go$(EXE_EXTENSION) build/toolchain/bin/protoc-gen-grpc-gateway$(EXE_EXTENSION) build/toolchain/bin/protoc-gen-swagger$(EXE_EXTENSION)
 
+install-stress-test-tools:
+	sudo apt-get install python3 python3-venv
```

Comment:
Use the virtualenv install that we already create. Don't assume users have python. See the python protobuf gen targets.

Reply:
Also just add the pip install to the python install you don't need a separate target.

Reply:
Our current Makefile doesnt have the python protobuf gen targets. I tried to install the virtualenv via `sudo apt-get install virtualenv python3-virtualenv` but it seems like nothing was installed into my bin directory. Virtualenv's official [installation guide](https://virtualenv.pypa.io/en/stable/installation/) seems like depends on either `pip` or `python`, that's why I explicitly list python3 as a requirement.

Reply:
https://github.com/GoogleCloudPlatform/open-match/commit/4cbee9d8a7f5ccbbee15222e56fbde176fc3710c#diff-b67911656ef5d18c4ae36cb6741b7965

It was removed recently add it back in.